src/main/java/UseCase/editPlaylist.java

```diff
@@ -10,7 +10,9 @@ public class editPlaylist implements editPlaylistInputBoundary{
      *                  and changeName()
      *
     */
```

Comment:
Initializer?

Reply:
Ignore this for now, I was going to correctly code it later